docs/reference/action_list.rst

```diff
@@ -761,5 +761,20 @@ query and displayed::
         $listMapper->addIdentifier('numberofcomments');
     }
 
+
+Lastly, you can also define your list fields as ``virtual``.
+This way, Sonata's FieldDescription will always return a value of null, as documented here:
+https://symfony.com/doc/current/bundles/SonataAdminBundle/cookbook/recipe_virtual_field.html ::
+
+    // src/Admin/PostAdmin.php
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper->addIdentifier('thisPropertyDoesNotExist', null, [
```

Comment:
I'm not sure `addIdentifier` should be use. How to use a virtual field as identifier ?
Isn't better to just use `add()` ?